lib/abao.coffee

```diff
@@ -5,6 +5,7 @@ chai = require 'chai'
 
 options = require './options'
 addTests = require './add-tests'
+testFactory = require './test'
```

Comment:
I think `TestFactory` is better, as I used Uppercase for class name through the project. While lowercase for functions.

Indeed, I just want to make sure the code style is consistent, what's your suggest, I'm open.